src/vs/workbench/contrib/terminal/browser/xterm/xtermTerminal.ts

```diff
@@ -159,7 +156,7 @@ export class XtermTerminal extends Disposable implements IXtermTerminal, IDetach
 	get shellIntegration(): IShellIntegration { return this._shellIntegrationAddon; }
 
 	get textureAtlas(): Promise<ImageBitmap> | undefined {
-		const canvas = this._webglAddon?.textureAtlas || this._canvasAddon?.textureAtlas;
+		const canvas = this._webglAddon?.textureAtlas;
 		if (!canvas) {
```

Comment:
what happens in this case?

Reply:
It'll work as before, just only for the webgl addon